aiida/orm/computers.py

```diff
@@ -590,7 +590,15 @@ def get_authinfo(self, user):
         """
         from . import authinfos
 
-        return authinfos.AuthInfo.objects(self.backend).get(dbcomputer_id=self.id, aiidauser_id=user.id)
+        try:
+            authinfo = authinfos.AuthInfo.objects(self.backend).get(dbcomputer_id=self.id, aiidauser_id=user.id)
+        except exceptions.NotExistent:
+            raise exceptions.NotExistent(
+                f'Computer `{self.label}` (ID={self.id}) not configured for user `{user.get_short_name()}` '
+                f'(ID={user.id}) - use `verdi computer configure ` first'
```

Comment:
```suggestion
                f'(ID={user.id}) - use `verdi computer configure` first'
```

Reply:
Question I am not sure of, and maybe @sphuber can help - should this `raise` be a `raise from` or it's not needed?

Reply:
it is not necessary, bu it is recommended, and in fact `pylint` should warn for this. So I wonder why that is not the case.

Reply:
Not sure what you mean by needed, but I agree it would be better.